scripts/install.js

```diff
@@ -46,50 +46,19 @@ var bar = createBar({ before: url + ' [' });
 
 var total = 0;
 var progress = 0;
-var d = download(url, dest, { extract: true, strip: 1 });
-d.on('response', function(res) {
-  total = parseInt(res.headers['content-length']);
-});
-d.on('data', function(data) {
-  progress += data.length;
-  if (total > 0) {
-    var percent = progress / total * 100;
-    bar.percent(percent);
-    if (percent >= 100) {
-      console.log('');
-      console.log('Extracting...');
-    }
-  }
-});
-d.on('error', error);
-d.on('close', function() {
-  // If OSX, manually set file permissions (until adm-zip supports getting the file mode from zips)
-  if (process.platform === 'darwin') {
-    if (!fs.existsSync(path.join(dest, 'Contents'))) {
-      dest = path.join(dest, 'nwjs.app');
+
+console.log( 'GET', url )
```

Comment:
Thought it'd be useful in the log output, in case something goes wrong - but sure, I can remove that.